recipe/meta.yaml

```diff
@@ -70,3 +70,4 @@ extra:
     - SylvainCorlay
     - bollwyvl
     - davidbrochart
+    - blink1073
```

Comment:
I'm willing to accept #62, created by @blink1073, as sufficent proof-of-interest.

Reply:
Confirmed, thanks!